app/config.js

```diff
@@ -15,6 +15,7 @@ export function configApp($urlRouterProvider, $httpProvider, localStorageService
       return LocalStorage.getJWT();
     },
   });
+  $httpProvider.interceptors.push('RedirectStatusInterceptor');
```

Comment:
it's better to register it as we register the agent interceptor in line 25 so we can share it with axios in `portainer/services/axios.ts`

Reply:
fixed